docs/goals-canvas.md.txt

```diff
@@ -0,0 +1,26 @@
+| Zoeken en raadplegen van exact die persoonsgegevens die nodig zijn voor  het uitvoeren van een wettelijke taak. | gericht zoeken op geboortedatum en geslachtsnaam + gevonden persoonsgegevens filteren op persoons- en/of relatiekenmerken | geboortedatum + (deel van) geslachtsnaam + optioneel naam- persoonskenmerkenfilter en/of naam-relatiekenmerken filter en optioneel indicatie "inclusief overleden personen". | verzameling (gefilterde) personen en/of relaties (zie onder) | Raadplegen van een collectie personen of het selecteren van een persoon uit de collectie om een wettelijke taak uit te voeren. | [447](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/issues/447){:target="_blank"} |
+| Zoeken en raadplegen van exact die persoonsgegevens die nodig zijn voor  het uitvoeren van een wettelijke taak. | gericht zoeken op geslachtsnaam, en woonplaatsgegevens + gevonden persoonsgegevens filteren op persoons- en/of relatiekenmerken | (deel van) geslachtsnaam + woonplaats + optioneel naam- persoonskenmerkenfilter en/of naam-relatiekenmerken filter en optioneel indicatie "inclusief overleden personen". | verzameling (gefilterde) personen en/of relaties (zie onder) | Raadplegen van een collectie personen of het selecteren van een persoon uit de collectie om een wettelijke taak uit te voeren. | |
```

Comment:
Wordt met woonplaatsgegevens verblijfplaatsgegevens bedoeld?

In de input is alleen woonplaats genoemd. Kan je alleen woonplaats (gemeente van inschrijving) of kan je met combinatie van verblijfplaatsgegevens (gemeenteVanInschrijving, straat, huisnummer, enz)

Reply:
Deze combinatie van parameters gaat uit van geslachtsnaam + woonplaats (=gemeente van inschrijving).